gef-ui/server/controllers/amendments/cover-end-date/get-cover-end-date.test.ts

```diff
@@ -0,0 +1,158 @@
+/* eslint-disable import/first */
+const getApplicationMock = jest.fn();
+const getFacilityMock = jest.fn();
+
+import * as dtfsCommon from '@ukef/dtfs2-common';
+import { aPortalSessionUser, DEAL_STATUS, Facility, PORTAL_LOGIN_STATUS, DEAL_SUBMISSION_TYPE, ROLES } from '@ukef/dtfs2-common';
+import { HttpStatusCode } from 'axios';
+import { createMocks } from 'node-mocks-http';
+import { getCoverEndDate, GetCoverEndDateRequest } from './get-cover-end-date';
+import { Deal } from '../../../types/deal';
+import { CoverEndDateViewModel } from '../../../types/view-models/amendments/cover-end-date-view-model';
+
+jest.mock('../../../services/api', () => ({
+  getApplication: getApplicationMock,
+  getFacility: getFacilityMock,
+}));
+
+const dealId = 'dealId';
+const facilityId = 'facilityId';
+const amendmentId = 'amendmentId';
+
+const companyName = 'test company';
+
+const getHttpMocks = () =>
+  createMocks<GetCoverEndDateRequest>({
+    params: { dealId, facilityId, amendmentId },
+    session: {
+      user: { ...aPortalSessionUser(), roles: [ROLES.MAKER] },
+      userToken: 'testToken',
+      loginStatus: PORTAL_LOGIN_STATUS.VALID_2FA,
+    },
+  });
+
+const mockDeal = { exporter: { companyName }, submissionType: DEAL_SUBMISSION_TYPE.AIN, status: DEAL_STATUS.UKEF_ACKNOWLEDGED } as Deal;
```

Comment:
same comment about the mockDeal as above please

Reply:
same here. I replaced with MOCK_BASIC_DEAL, MOCK_ISSUED_FACILITY, MOCK_UNISSUED_FACILITY. Could you check again please?